deploy/images.libsonnet

```diff
@@ -0,0 +1,15 @@
+// last updated: 1/28/2019
+{
+    _images+:: {
+        alertmanager: "quay.io/cortexproject/alertmanager:master-0ef1ce95",
```

Comment:
You should make the master branch tag a local var so it doesn't have be changed in so many places to update everything at once